app/models/certificate.rb

```diff
@@ -23,4 +23,10 @@ def x509
   def issuer_common_name
     x509.subject.to_a.find { |issuer, _, _| issuer == 'CN' }[1]
   end
+
+  def certificate_expiry
```

Comment:
this probably should be called `expires_soon` or `days_left` ... it might be confusing for anyone wishing to use the method to obtain the expiry date but will get it only if it's less than 30 days.

We should consider refactoring this to a `status` instead, given we have 4 tags (IN USE, EXPIRES IN x DAYS, DEPLOYING, MISSING). What about a method which could determine the actual status in the model? would save some of the `if` branching in the view...

Reply:
Should i create a follow up story or just implement as part of this?